src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    private final Random random = new Random();
```

Comment:
Please don’t add redundant empty lines to your code.
We don’t need them after class declaration or method signature.